ntpcheck/cmd/diag.go

```diff
@@ -122,6 +123,18 @@ func checkJitter(r *checker.NTPCheckResult) (status, string) {
 	)
 }
 
+func checkCorrectionMetric(r *checker.NTPCheckResult) (status, string) {
+	const warnThreshold time.Duration = time.Minute
+	const failThreshold time.Duration = 10 * time.Minute
+	return checkAgainstThreshold(
+		"Current correction",
+		r.Correction,
+		float64(warnThreshold.Milliseconds()),
+		float64(failThreshold.Milliseconds()),
+		"Correction is the .",
```

Comment:
That's what `man chronyc` section about 'tracking' results says:

```
        System time
               In normal operation, chronyd by default never steps the system clock, because any jump in the time can have adverse consequences for
               certain application programs. Instead, any error in the system clock is corrected by slightly speeding up or slowing down the system
               clock until the error has been removed, and then returning to the system clock’s normal speed. A consequence of this is that there will
               be a period when the system clock (as read by other programs) will be different from chronyd’s estimate of the current true time (which
               it reports to NTP clients when it is operating in server mode). The value reported on this line is the difference due to this effect.
```


I think we can put here something like this: 'Correction is the difference between system time and chronyd’s estimate of the current true time.'